applications/FemToDemApplication/custom_processes/assign_pressure_id_process.h

```diff
@@ -47,7 +47,8 @@ class AssignPressureIdProcess : public Process
 protected:
 
   // Member Variables
-  ModelPart &mr_model_part;
+  ModelPart &mrModelPart;

```

Comment:
```suggestion
  ModelPart& mrModelPart;
```

I am quite sure we use type& thing instead of type &thing